### ISSUE TITLE
zstd-sys should use the BSD 3-clause license

### ISSUE THREAD

**Issue by complexmath** [gyscos/zstd-rs#314]
I see that the BSD 3-clause license that the zstd library uses was added to the zstd-sys tree here: https://github.com/gyscos/zstd-rs/issues/171. However, the zstd-sys LICENSE file says that the package uses the MIT or Apache licenses with no mention of the BSD license despite all the generated code in the project using the BSD license. I think the zstd-sys package should be changed to using only the BSD license to reflect this, and so the license is represented properly in the zstd-sys crate. Since the 3-clause license has a documentation requirement, it seems highly likely that with the current setup, many users of zstd are in violation of the license requirements simply from not knowing that the BSD license is involved.

**Comment by Kasifrasi** [comment]
Running into this exact issue when using cargo-about for compliance scans. The missing BSD-3-Clause in the Cargo.toml causes strict SPDX validators to fail or miss the bundled C-code license. A fix would be highly appreciated!

**Comment by jhult** [comment]
Related: https://github.com/gyscos/zstd-rs/pull/358

**Comment by gyscos** [comment]
Yes, indeed - the current MIT / apache is just leftover from the very early time of the lib, before zstd-sys was its own crate. I want to make the lib just as permissive as zstd itself, so let's copy their "BSD or GPLv2" license (at least for the bindings).

**Comment by jhult** [comment]
BSD-3-Clause is more permissive than GPL-2.0. To that effect, I just updated everything to use only the BSD-3-Clause in this PR: https://github.com/gyscos/zstd-rs/pull/360

**Comment by jhult** [comment]
@complexmath and @gyscos, I believe this can now be closed.
